src/components/application_manager/include/application_manager/hmi_capabilities_impl.h

```diff
@@ -78,6 +78,10 @@ class HMICapabilitiesImpl : public HMICapabilities {
 
   void set_ccpu_version(const std::string& ccpu_version) OVERRIDE;
 
+  bool matches_ccpu_version(const std::string& ccpu_version) OVERRIDE;
```

Comment:
Why non const?

Reply:
@LuxoftAKutsan [done](https://github.com/smartdevicelink/sdl_core/pull/3269/files#diff-5655615f464bf3d9794062f13d8749caR81)

Reply:
@ydementieiev Not Done! 
@LuxoftAKutsan  did mean different thing. This thing is not "return by const value", but const function.
Returning by const value has no sense...
```cpp
 bool matches_ccpu_version(const std::string& ccpu_version) const OVERRIDE;
```